js/libs/form.js

```diff
@@ -639,8 +639,7 @@ Fliplet.Widget.instance('form-builder', function(data) {
 
           $vm.loadEntryForUpdate();
         }, function(err) {
-          console.error(err);
```

Comment:
why did you remove the `console.error`?

Reply:
I took a look at other catch structures in widget, and found out that we don't use console there except one place in `image.js` which was added 4 years ago. Also I have a habit to clear consoles before delivering tasks.
If I it is useful for testing/Sentry I'll bring it back now.

Reply:
Please bring it back, this is useful when clients and their developers are debugging via Fliplet Studio

Reply:
Done.